packages/amplify-e2e-core/src/utils/rds.ts

```diff
@@ -308,24 +371,29 @@ export const setupRDSInstanceAndData = async (
  * @returns Endpoint address, port and database name of the created RDS cluster.
  */
 
-export const setupRDSClusterAndData = async (config: RDSConfig, queries?: string[]): Promise<ClusterInfo> => {
-  console.log(`Creating RDS ${config.engine} DB cluster with identifier ${config.identifier}`);
+export const setupRDSClusterAndData = async (useLocalCluster: boolean, config: RDSConfig, queries?: string[]): Promise<ClusterInfo> => {
```

Comment:
Continuing the above discussion of `clusterInfo` as global state -- this method will set the `clusterInfo` global as a side effect, but it doesn't need to -- the method actually returns cluster info. Consider removing the global and making callers responsible for retaining the value so they can pass it to subsequent calls that need it.